lib/webhooks/types.ts

```diff
@@ -12,6 +12,7 @@ export type WebhookHandlerFunction = (
   shop_domain: string,
   body: string,
   webhookId: string,
+  apiVersion: string,
```

Comment:
We should make this optional (just for the type) so this isn't a breaking change for TS apps.